README.md

```diff
@@ -1,6 +1,6 @@
 # adapt-scrollPrompt
 
-A button which can be appended to menu and/or page headers.
+A button which can be appended to menu, page headers, and/or components.
```

Comment:
In all instances, the button is appended to the header.

Reply:
Done in [775b1b5](https://github.com/cgkineo/adapt-scrollPrompt/pull/4/commits/775b1b578626b42bcb85520f605a1a84f019cec5)